### PR TITLE
Update GameStatus fields

### PR DESCRIPTION
**Pull request by HailSanta**
names for a few fields in game status

### CODE REVIEW

**Review by Brotenko — APPROVED** [review]
LGTM to me

File: include/common_structs.h [hunk 1]
```diff
@@ -1204,30 +1204,30 @@ typedef struct GameStatus {
     /* 0x0A8 */ s8 creditsViewportMode;
     /* 0x0A9 */ s8 unk_A9;
     /* 0x0AA */ s8 demoFlags;
-    /* 0x0AB */ u8 unk_AB;
-    /* 0x0AC */ s8 loadMenuState;
-    /* 0x0AD */ s8 menuCounter;
+    /* 0x0AB */ u8 usingStereoSound;
```
Comment (nanaian) [inline include/common_structs.h:1207]: If this is a bool I'd prefix with `is`...
```suggestion
    /* 0x0AB */ u8 isUsingStereoSound;
```
Or
```suggestion
    /* 0x0AB */ u8 isStereo;
```

File: src/rumble.c [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 // TODO: replace nustuff with defines
 
-u16 D_80074260 = 0;
-s32 D_80074264 = 0;
+u16 RumbleMaxDuration = 0;
+s32 RumbleButtons = 0;
```
Comment (nanaian) [inline src/rumble.c:7]: These should be lowerCamelCase, and possibly prefixed like `rumble_maxDuration` and `rumble_buttons`

**Review by nanaian — CHANGES_REQUESTED** [review]

**Review by ethteck — APPROVED** [review]
Happy after the existing stuff is addressed. Thanks :)

File: src/rumble.c (hunk 2 above)
Comment (nanaian) [reply src/rumble.c:7]: @ethteck do you think we should prefix it like rumble_ or just rumbleMaxDuration

**Comment by BowserSlug** [comment]
✅ This PR fixes 49 warnings!

File: src/165490.c [hunk 3]
```diff
@@ -56,7 +56,7 @@ void filemenu_draw_contents_stereo(
     s32 width, s32 height,
     s32 opacity, s32 darkening
 ) {
-    if (gGameStatusPtr->usingStereoSound == 1) {
+    if (gGameStatusPtr->isUsingStereoSound == 1) {
```
Comment (nanaian) [inline src/165490.c:59]:
```suggestion
    if (gGameStatusPtr->isUsingStereoSound) {
```

File: src/165490.c [hunk 4]
```diff
@@ -71,7 +71,7 @@ void filemenu_draw_contents_mono(
     s32 width, s32 height,
     s32 opacity, s32 darkening
 ) {
-    if (gGameStatusPtr->usingStereoSound == 0) {
+    if (gGameStatusPtr->isUsingStereoSound == 0) {
```
Comment (nanaian) [inline src/165490.c:74]:
```suggestion
    if (!gGameStatusPtr->isUsingStereoSound) {
```

File: src/state_startup.c [hunk 5]
```diff
@@ -72,10 +72,10 @@ void state_step_startup(void) {
     fio_has_valid_backup();
 
     if (D_800D9620 == 0) {
-        gGameStatusPtr->usingStereoSound = 1;
+        gGameStatusPtr->isUsingStereoSound = 1;
```
Comment (nanaian) [inline src/state_startup.c:75]:
```suggestion
        gGameStatusPtr->isUsingStereoSound = TRUE;
```

File: src/state_startup.c [hunk 6]
```diff
@@ -72,10 +72,10 @@ void state_step_startup(void) {
...
         func_8005615C();
     } else {
-        gGameStatusPtr->usingStereoSound = 0;
+        gGameStatusPtr->isUsingStereoSound = 0;
```
Comment (nanaian) [inline src/state_startup.c:78]:
```suggestion
        gGameStatusPtr->isUsingStereoSound = FALSE;
```

**Review by nanaian — CHANGES_REQUESTED** [review]
Sorry for not catching these before

**Pull request merged**